handler/utils.go

```diff
@@ -18,3 +18,18 @@ func filterPopulationTypes(coundition []string, toFilter []contract.PopulationTy
 
 	return filtered
 }
+
+type CantabularError string
+
+const (
+	MaxVariableError = "Maximum variables at MSOA and above is 5"
```

Comment:
Where is this text? Local query when run with more than 5 variables returns `Maximum variables in query is 5`

Reply:
My local gives that error message